### DISCUSSION TITLE
Location verification not working on Mac

### DISCUSSION THREAD

**Discussion by AmeliaMN** [community/community#101449] [GitHub Education] [answered]
### Hi everyone,

I opened a support ticket 3 weeks ago on this issue, but haven't heard anything back. I'll just copy-paste the information here, in case this is a better method to get a response. 

**Product**

GitHub

**Description**

I'm trying to re-verify my educator status for GitHub education. The form requires location services to be enabled, to verify you are close to your institution. I've tried both Firefox (119.0.1 (64-bit)) and Chrome (120.0.6099.129 (Official Build) (arm64)) on my Mac, and neither will work. On both browsers, I've tried enabling location services before I go to the form, enabling only when asked, reloading the page repeatedly, and clearing cookies and cache. Even with location enabled, I still get the "To continue you must modify your browser settings to allow GitHub to use your location information." message.

Is there a workaround?

<img width="1485" alt="Screen Shot 2024-01-04 at 11 04 40 AM" src="https://github.com/community/community/assets/2576787/335b5ea2-8d3d-404a-a13b-b731f0dc4776">
<img width="1580" alt="Screen Shot 2024-01-04 at 11 04 55 AM" src="https://github.com/community/community/assets/2576787/d0d8359a-7946-4ec5-99f3-ef1bf4cb32fa">

**Reproduction Steps**

Go to [the GitHub for education form](https://education.github.com/discount_requests/application). Enable location services on a Mac. Attempt to submit form. Fail. Re-load page. Attempt to submit form. Fail. Clear cookies and cache, reload page, re-enable location services, attempt to submit form. Fail. Etc.

**Fault Type**

Permanent - Occurring repeatably

**Blocking?**

Yes

**Comment by AmeliaMN** [comment]
I just tried the form on my phone (Pixel 4a, Android version 13) on Firefox (121.1.0) and was able to get past the location verification step. However, the next step of uploading proof of academic status then didn't work. The phone camera wouldn't switch to my back camera, so I uploaded a photo instead. Then I pressed the "Process my application" button and it greyed out and said "Processing information..." It got stuck like that for more than 5 minutes, and then I tried again, with the same result.

**Comment by AmeliaMN** [comment, marked as answer]
In case anyone runs across this searching for answers, the "solution" I finally got from GitHub support was basically, try other computers. On my work desktop (running Mac OS 13.3.1, Ventura) I was able to get the location verification step to work on Google Chrome (Version 121.0.6167.160 (Official Build) (x86_64)). However, then the file upload page wouldn't work. The support agent suggested not using the file upload feature but instead taking a photo directly. That worked. It's a little ridiculous that I had to try 3+ devices and 2x3=6+ browsers with different configurations before I was able to find one that worked.

**Reply by jaredvt** [reply]
Booo....
